pkgs/build-support/rust/build-rust-crate/configure-crate.nix

```diff
@@ -139,13 +139,8 @@ in ''
        | sed -e "s/cargo:\([^=]*\)=\(.*\)/export DEP_$(echo $CRATENAME)_\U\1\E=\2/" > target/env
 
      set -e
-     if [[ -n "$(ls target/build/${crateName}.out)" ]]; then
```

Comment:
I don't think we have to keep the libPath version of this. This would just cause the same issue for any crate that has a custom path to the `lib.rs` file configured.